requests_ntlm/requests_ntlm.py

```diff
@@ -55,11 +55,19 @@ def retry_using_http_NTLM_auth(self, auth_header_field, auth_header,
         request = response.request.copy()
 
         # initial auth header with username. will result in challenge
-        msg = "%s\\%s" % (self.domain, self.username) if self.domain else self.username
+        if not self.domain:
+            msg = self.username
+            type1_flags = ntlm.NTLM_TYPE1_FLAGS & \
+                ~ntlm.NTLM_NegotiateOemDomainSupplied
+
+        else:
+            msg = "%s\\%s" % (self.domain, self.username)
+            type1_flags = ntlm.NTLM_TYPE1_FLAGS
 
         # ntlm returns the headers as a base64 encoded bytestring. Convert to
         # a string.
-        auth = '%s %s' % (auth_type, ntlm.create_NTLM_NEGOTIATE_MESSAGE(msg).decode('ascii'))
+        auth = '%s %s' % (auth_type, ntlm.create_NTLM_NEGOTIATE_MESSAGE(
+            msg, type1_flags).decode('ascii'))
```

Comment:
At this point I think we're making our life difficult by trying to jam so much into one line. Let's pull `ntlm.create_NTLM_NEGOTIATE_MESSAGE(msg, type1_flags).decode('ascii')` out of this line and into its own line, and assign it to the temporary variable `negotiate_message`. That should help clear this up a little bit.